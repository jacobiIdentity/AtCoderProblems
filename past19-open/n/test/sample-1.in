7
3 1 4 1 5 9 2
0.25
4
1 3
3 6
1 7
2 2
