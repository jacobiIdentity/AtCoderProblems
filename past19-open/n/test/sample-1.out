3.5
4.703125
3.54443359375
1
